v20.2/spatial-data.md

```diff
@@ -0,0 +1,305 @@
+---
+title: Working with Spatial Data
+summary: CockroachDB has special support for efficiently storing and querying spatial data.
+toc: true
+---
+
+<span class="version-tag">New in v20.2</span>: CockroachDB has special support for efficiently storing and querying spatial data. Supported spatial data types include:
+
+- Geometric objects such as points, lines, paths, and polygons in 2-dimensional space. These are projected onto the flat surface of a plane and are represented in SQL by the `GEOMETRY` data type.
+
+- Geographic objects, which are also made up of points, lines, polygons, etc., in 2-dimensional space. They are projected onto the surface of a sphere and are represented in SQL by the `GEOGRAPHY` data type. (Technically, they are projected onto a spheroid: "a sphere with a bulge"). This projection means that paths between geographic objects are not straight lines; they are curves, and so the distances between objects are calculated using [great circle math](https://en.wikipedia.org/wiki/Great-circle_distance).
```

Comment:
I would try and somehow word "Long/Lat" as the "x,y" coordinate representations here.